deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml

```diff
@@ -1,29 +1,20 @@
 receivers:
-  filelog/dockercontainers:
-    include: [  "/var/lib/docker/containers/*/*.log" ]
-    start_at: end
-    include_file_path: true
-    include_file_name: false
+  tcplog/docker:
```

Comment:
Just curious, how is syslog with tcp config `syslog::tcp::isten_address::` different from this.

Reply:
Not much difference,  here we just have the explicit regex.